src/marshmallow/fields.py

```diff
@@ -468,8 +468,17 @@ def schema(self):
             # Inherit context from parent.
             context = getattr(self.parent, "context", {})
             if isinstance(self.nested, SchemaABC):
-                self._schema = self.nested
+                self._schema = copy.deepcopy(self.nested)
```

Comment:
I have nested schema instance which context contains non-pickling object. In my case it is `Crypto.Cipher.AES` object which i tried to deepcopy in console and got an error.

Reply:
Does it fix it if you use `copy.copy` instead?

```suggestion
                self._schema = copy.copy(self.nested)
```

Now that I think of it, I'm not sure we even need to deepcopy here. Let me know if the above fixes your issue.

Reply:
@sloria yep, copy works just fine! waiting for release, thanks!

Reply:
Thanks for checking. The fix is released in 3.1.1.